src/pages/theEyePages/accountList/accountList.css

```diff
@@ -0,0 +1,40 @@
+.account-list{
```

Comment:
Might be worth converting these CSS files into JSS (https://cssinjs.org/?v=v10.9.0) once we get the chance to keep consistency with other  parts of the project. But overall looks good!